project-plan.md

```diff
@@ -0,0 +1,189 @@
+
+**2. Todo**
+
+List of tasks that are planned to be implemented in the near future, and have enough details ironed out to make it possible to start the implementation.
+
+**3. In Progress**
+
+List of tasks that are in progress. A developer have written words or code. These tasks should ideally be linked to an open Pull Request on Github with a WIP label on it.
+
+**4. In Review**
+
+List of tasks where the developer has implemented the task, and is waiting to get a stamp of approval from a second pair of eyes. Approval could be given by the client, supervisor or any of the domain experts. The developer will request review from the specific person which is considered best suited to review the pull request. This specific person will be notified about this on Discord, and via email if necessary.
+
+**5. Done**
+
+List of task where the pull request linked to the task has been approved in merged into the main branch of the git source tree.
+
+## 4.2 Plan for statusmøter og beslutningspunkter i perioden
+
+We have planned 2 weekly meetings, 1 with supervisor and 1 with client and domain experts.
+
+- Client and Domain experts meeting: Tuesdays @ 12:30
+- Academic supervisor meeting: Tuesdays @ 13:00
+
+Participants of these meetings have received invites by email, to make it easy to add the scheduled meetings to their respective calendars.
+
+# 5. Organisering av kvalitetssikring
+
+## 5.1 Dokumentasjon, standardbruk og kildekode
+
+The language used to develop the proof-of-concept app as not been decided yet, but we would like to use a language with mature tools for documentation, testing, linting, formatting. These tools should be run by the CI on every commit to the source-tree to ensure consistent quality.
+
+Here are some key areas of quality control, which can be expected:
+
+### API documentation generated from code docstrings
+
+This is a very useful tool to document interfaces in the code bases. Either interfaces between different modules in the code-base, or external interfaces like a CLI or HTTP API etc.
+
+### High level visual documentation
+
+Some high-level documentation will be required to explain the architecture of the entire solution, using diagrams.
+
+### Testing of API's
+
+There will be an interface to control the proof-of-concept app. Most of the testing we do should focus heavily on testing this interface, be it a CLI or a HTTP API or both or something else. How this interface will look is TBD.
+
+**@TODO: Define API of proof-of-concept app**
+
+### Linting and formatting using community defaults and best practices
+
+Programming communities have standards and default practices regarding how to write code. Automated tools like a linter should run in the CI environment to ensure that these best practices are being followed.
+
+
+## 5.2 Konfigurasjonsstyring
+
+**@TODO**
+
+## 5.3 Risikoanalyse (identifisere, analysere, tiltak, oppfølging)
+
+### Risk 1: Proof-of-concept app development-cost unpredictability
```

Comment:
Should we define actionpoints if the risk becomes clear?

Reply:
Yes. That is definitely a **@TODO** :+1: